coremltools/models/neural_network/builder.py

```diff
@@ -2271,39 +2351,46 @@ def add_convolution(
 
         dilation_factors: list of int
             Dilation factors across height and width directions. Must be a list of two positive integers.
-            Defaults to [1, 1]
+            Defaults to ``[1, 1]``.
 
         padding_top, padding_bottom, padding_left, padding_right: int
-            values of height (top, bottom) and width (left, right) padding to be used if border_more is "valid".
+            values of height (top, bottom) and width (left, right) padding 
+            to be used if ``border_more`` is ``"valid"``.
 
-        same_padding_asymmetry_mode: str.
-            Type of asymmetric padding to be used when  border_mode is 'same'.
-            Can be either 'BOTTOM_RIGHT_HEAVY' or  'TOP_LEFT_HEAVY'.
-
-        Depthwise convolution is a special case of convolution, where we have:
-            kernel_channels = 1 (== input_channels / groups)
-            output_channels = channel_multiplier * input_channels
-            groups = input_channels
-            W: [Kernel_height, Kernel_width, 1, channel_multiplier * input_channels]
+        same_padding_asymmetry_mode: str
+            Type of asymmetric padding to be used when  ``border_mode`` is ``'same'``.
+            Can be either ``'BOTTOM_RIGHT_HEAVY'`` or  ``'TOP_LEFT_HEAVY'``.
 
+		Depthwise convolution
+			Depthwise convolution is a special case of convolution, in which:
+			
+            	* ``kernel_channels = 1 (== input_channels / groups)``
+            	* ``output_channels = channel_multiplier * input_channels``
+           	* ``groups = input_channels``
+            	* ``W``: ``[Kernel_height, Kernel_width, 1, channel_multiplier * input_channels]``
 
-        Quantization arguments expected in kwargs, when W is of type bytes():
+		Quantization
+			Quantization arguments expected in ``kwargs``, when ``W`` is of type ``bytes()``.
 
-            quantization_type: str
-                When weights are quantized (i.e. W is of type bytes()), this should be either "linear" or "lut".
+				quantization_type: str
+					When weights are quantized (that is, ``W`` is of type ``bytes()``), 
+					this should be either ``"linear"`` or ``"lut"``.
 
-            nbits: int
-                Should be between 1 and 8 (inclusive). Number of bits per weight value. Only applicable when
-                weights are quantized.
+				nbits: int
```

Comment:
Looks like the indentation is wrong here. There is probably mixed usage of spaces and tabs. It might even look right on your screen.

Reply:
Actually done on purpose, so that these parameters would appear indented under `Quantization`. Same indenting with the `Depthwise Convolution` section above it. The current formatting is obviously not right, so this is my attempt to fix it. Here's how it will look with the new formatting -- please look it over to see if it is correct:

![Screen Shot 2021-10-06 at 6 43 59 AM](https://user-images.githubusercontent.com/70229264/136247582-702ca27e-7224-49c5-8d7d-f30dc161ea99.png)